src/cli/cli_coap_secure.cpp

```diff
@@ -943,7 +943,7 @@ void CoapSecure::HandleResponse(otMessage *aMessage, const otMessageInfo *aMessa
     {
         OutputLine("coaps receive response error %d: %s", aError, otThreadErrorToString(aError));
     }
-    else
+    else if((aMessageInfo != nullptr) && (aMessage != nullptr))
```

Comment:
If I am not mistaken, when `aError` is `OT_ERROR_NONE` we expect `aMessage` to be provided (should not be `nullptr`).

Reply:
I submitted https://github.com/openthread/openthread/pull/11287 to remove the unnecessary `nullptr` checks. Thanks.